rust/arrow/src/compute/kernels/comparison.rs

```diff
@@ -210,6 +210,10 @@ where
     T: ArrowNumericType,
     F: Fn(T::Simd, T::Simd) -> T::SimdMask,
 {
+    use crate::buffer::MutableBuffer;
```

Comment:
Our convention has been to put imports at the beginning of the file. I can see why you put them here, so it's fine.

Reply:
yes, I did try putting the imports at the beginning of the file but this generates warnings when the simd feature is disabled (because these imports are only used for that feature); so I thought that to put the imports inside the function would be the most efficient way around this

Reply:
Yea, I figured, because you'd have to otherwise feature-gate them.